### DISCUSSION TITLE
;oad xml function does not work

### DISCUSSION THREAD

**Discussion by Ho-ChunHuang-NOAA** [dtcenter/METplus#1559] [Plot Generation] [answered]
Hi,

I can not load users xml up to https://metviewer.nws.noaa.gov/servlet?jsp=new.

I first save the xml file to the local disk,. plot_20220407_142444, then I try "load xml".  It does nothing, no error message of failure, but an new field with null entry.

Please take a look.

**Comment by TatianaBurek** [comment]
I can't reproduce the issue. I was able to load XML from plot_20220407_142444 and successfully create a new Taylor diagram.
Try to use this URL: https://metviewer.nws.noaa.gov/ instead of the one that you put into the comment

**Comment by TatianaBurek** [comment]
@Ho-ChunHuang-NOAA Do you still have a problem with loading XML?

**Comment by Ho-ChunHuang-NOAA** [comment, marked as answer]
Hi, Tatiana:

I remain not able to load the XML to
https://metviewer.nws.noaa.gov/metviewer1.jsp.  But please close the
ticket.  I can live with it.

Ho-Chun Huang, Ph.D.

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958
